charts/prometheus-mysql-exporter/values.yaml

```diff
@@ -8,7 +8,8 @@ fullnameOverride: ""
 replicaCount: 1
 
 image:
-  repository: "prom/mysqld-exporter"
+  registry: quay.io
+  repository: prometheus/mysqld-exporter
```

Comment:
when we change default of an existing variable we should raise major version and add some info in the upgrading section of the chart.

Reply:
I modelled partially after #928 which did basically the same for the `alertmanager` chart in a minor bump. The adding of the `registry` key does make this slightly different though. You really think a major version bump is the way to go here? Then I'll add some notes to the `README`.